Description: Adds /usr/lib/rbenv/hooks to the hook path
 This patch adds a new directory to the hook path that is suitable for rbenv
 plugins packaged in Debian.
 .
 This patch was already applied upstream, so it will be dropped in the next
 upstream release.
Author: Antonio Terceiro <terceiro@debian.org>
Origin: vendor
Bug: https://github.com/sstephenson/rbenv/pull/183

--- rbenv-0.3.0.orig/libexec/rbenv
+++ rbenv-0.3.0/libexec/rbenv
@@ -48,7 +48,7 @@ for plugin_bin in "${RBENV_ROOT}/plugins
 done
 export PATH="${bin_path}:${PATH}"
 
-hook_path="${RBENV_HOOK_PATH}:${RBENV_ROOT}/rbenv.d:/usr/local/etc/rbenv.d:/etc/rbenv.d"
+hook_path="${RBENV_HOOK_PATH}:${RBENV_ROOT}/rbenv.d:/usr/local/etc/rbenv.d:/etc/rbenv.d:/usr/lib/rbenv/hooks"
 for plugin_hook in "${RBENV_ROOT}/plugins/"*/etc/rbenv.d; do
   hook_path="${hook_path}:${plugin_hook}"
 done
